<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Secret Keeper Router</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="gradient">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>540</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/gradient.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="connect_info_button">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>19</y>
      <width>321</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
 background: rgba(255, 255, 255, 0.05);
 border-radius: 8px;
 color: #fff;
}
QPushButton:hover{
 background: rgba(255, 255, 255, 0.1);
}
QPushButton:pressed{
 background: rgba(0, 0, 0, 0.1);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="all_connections_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
 background: rgba(255, 255, 255, 0.05);
 border-radius: 8px;
 color: #fff;
}
QPushButton:hover{
 background: rgba(255, 255, 255, 0.1);
}
QPushButton:pressed{
 background: rgba(0, 0, 0, 0.1);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="data_change_router_button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
 background: rgba(255, 255, 255, 0.05);
 border-radius: 8px;
 color: #fff;
}
QPushButton:hover{
 background: rgba(255, 255, 255, 0.1);
}
QPushButton:pressed{
 background: rgba(0, 0, 0, 0.1);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="initial_settings_button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
 background: rgba(255, 255, 255, 0.05);
 border-radius: 8px;
 color: #fff;
}
QPushButton:hover{
 background: rgba(255, 255, 255, 0.1);
}
QPushButton:pressed{
 background: rgba(0, 0, 0, 0.1);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="data_change_wifi_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
 background: rgba(255, 255, 255, 0.05);
 border-radius: 8px;
 color: #fff;
}
QPushButton:hover{
 background: rgba(255, 255, 255, 0.1);
}
QPushButton:pressed{
 background: rgba(0, 0, 0, 0.1);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="text_1_1">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>90</y>
      <width>290</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#e3eded</string>
    </property>
    <property name="text">
     <string>Изменение данных</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_1_2">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>140</y>
      <width>70</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#e3eded</string>
    </property>
    <property name="text">
     <string>WIFI</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_2_1">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>90</y>
      <width>290</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#e3eded</string>
    </property>
    <property name="text">
     <string>Изменение данных</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_2_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#e3eded</string>
    </property>
    <property name="text">
     <string>роутера</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_3_1">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>350</y>
      <width>170</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#e3eded</string>
    </property>
    <property name="text">
     <string>Показ всех</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_4_1">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>350</y>
      <width>310</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#e3eded</string>
    </property>
    <property name="text">
     <string>Сброс до начальных</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_3_2">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>400</y>
      <width>210</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#e3eded</string>
    </property>
    <property name="text">
     <string>подключений</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_4_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>400</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#e3eded</string>
    </property>
    <property name="text">
     <string>настроек</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
